app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -197,18 +226,8 @@ public function getCartDataJs($checkoutType = self::CHECKOUT_TYPE_MULTI_PAGE)
 
             // Call Bolt create order API
             try {
-                /////////////////////////////////////////////////////////////////////////////////
-                // We create a copy of the quote that is immutable by the customer/frontend
-                // Bolt saves this quote to the database at Magento-side order save time.
-                // This assures that the quote saved to Magento matches what is stored on Bolt
-                // Only shipping, tax and discounts can change, and only if the shipping, tax
-                // and discount calculations change on the Magento server
-                ////////////////////////////////////////////////////////////////////////////////
-                /** @var Mage_Sales_Model_Quote $immutableQuote */
-                $immutableQuote = $boltHelper->cloneQuote($sessionQuote, $checkoutType);
-                ////////////////////////////////////////////////////////////////////////////////
-
-                $orderCreationResponse = $this->getBoltOrderToken($immutableQuote, $checkoutType);
+
+                $orderCreationResponse = $this->getBoltOrderToken(null, $checkoutType);
```

Comment:
- why not directly call `getBoltOrderTokenPromise`?
- variable should be named something like promise

Reply:
Good point.  I probably was to caught up in the idea of preserving legacy interfaces.  I will update to a direct call and remove the `null` condition.

For the variable name, due to that it can be something other than a Promise, perhaps using the BoltCheckout name for this, `cartData` is the most appropriate.